scripts/xdp-devkit.json

```diff
@@ -1,3 +1,3 @@
 {
-    "path" : "https://github.com/microsoft/xdp-for-windows/releases/download/v0.16.8/xdp-devkit-x64-0.16.8-prerelease.zip"
+    "path" : "https://github.com/microsoft/xdp-for-windows/releases/download/v0.17.0/xdp-devkit-x64-0.17.0.zip"
```

Comment:
Not a comment on this PR, but should the release still be named prerelease so long as there are no signed binaries, or have we just moved away from that naming convention in XDP?

Reply:
Right, we're moving away from that notion. The -prerelease tag was never meant to be in the release titles; the 0 major series is implicitly prerelease. The -prerelease tag was meant for builds in e.g. main.